src/partners/kyber/config/config.js

```diff
@@ -1,23 +1,35 @@
 // TODO: look into only using kyberTokenInfoList for ETH and parsing results differently if ROP?
 import { TIME_SWAP_VALID_DEFAULT } from '../../partnersConfig/config';
 
+import { KyberCurrenciesETH } from './currenciesETH';
+import { KyberCurrenciesROP } from './currenciesROP';
+// const ETH_ENDPINT = 'https://tracker.kyber.network';
```

Comment:
can someone please remove @SteveMieskoski 's comments

Reply:
got it